tutorial/output-cache-open-ai/infra/shared/apps-env.bicep

```diff
@@ -0,0 +1,33 @@
+param name string
+param location string = resourceGroup().location
```

Comment:
If we onboard new tutorials requiring the same infra setup, let us move this folder to a common place to avoid duplicate.

Reply:
@yugangw-msft  I don't think the infra folder should be shared - it's meant for a per-project deployment. i.e. anyone copying over a folder would be able to run the project with the bicep files. 
Let me know if I misunderstood anything.

Reply:
I meant the bicep files to create a Redis Enterprise inside a VNET will be same across all the templates, correct? Efforts are worthwhile to consider so to reduce the duplicate to make sure future update will be simpler.